src/pydio/res/i18n/it.po

```diff
@@ -6,42 +6,43 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: 1\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-05-26 15:18+0200\n"
-"PO-Revision-Date: 2016-05-26 15:59+0200\n"
-"Last-Translator: thomas saliou <thomas@pydio.com>\n"
+"PO-Revision-Date: 2016-06-24 12:36+0100\n"
+"Last-Translator: DepaMarco <depardimarco@hotmail.it>\n"
 "Language-Team: Italian\n"
 "Language: it\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.5.4\n"
 
 #: job/change_processor.py:160
 #, python-format
 msgid "New folder created at %s"
-msgstr "Nuova cartella creata al %s"
+msgstr "Nuova cartella creata il %s"
 
 #: job/change_processor.py:166 job/change_processor.py:173
 #, python-format
 msgid "Folder created at %s"
-msgstr "Cartella creata al "
+msgstr "Cartella creata il %s"
 
 #: job/change_processor.py:185
 #, python-format
 msgid "Deleted file %s"
-msgstr "File %s sopresso"
+msgstr "File %s eliminato"
 
 #: job/change_processor.py:191
 #, python-format
 msgid "Folder %s deleted"
-msgstr "Cartella %s sopressa"
+msgstr "Cartella %s eliminato"
```

Comment:
My fault! :( sorry... the right one is "eliminata" :) I commit the change to this with the other strings